src/features/profile/pages/TrackEmailChange/TrackEmailChange.tsx

```diff
@@ -52,29 +65,38 @@ export function TrackEmailChange() {
           <Step>
             <StyledStepCard
               type={getStepCardType(0)}
-              title="Envoie ta demande"
+              title="Envoi de ta demande"
               icon={<BicolorPhoneIcon />}
             />
           </Step>
           <Step>
             <StyledStepCard
               type={getStepCardType(1)}
-              title="Confirme ta demande"
-              subtitle={currentStep === 1 ? `Depuis l’email envoyé à ${currentEmail}` : undefined}
+              title={currentStep === 1 ? 'Confirme ta demande' : 'Confirmation de ta demande'}
+              subtitle={`Depuis l’email envoyé à ${currentEmail}`}
               icon={<BicolorEmailIcon />}
             />
           </Step>
           <Step>
             <StyledStepCard
               type={getStepCardType(2)}
-              title="Validation de ta nouvelle adresse"
+              title={
+                currentStep === 2
+                  ? 'Valide ta nouvelle adresse'
+                  : 'Validation de ta nouvelle adresse'
+              }
+              subtitle={`Depuis l’email envoyé à ${newEmail}`}
               icon={<BicolorEmailIcon />}
             />
           </Step>
           <Step>
             <StyledStepCard
               type={getStepCardType(3)}
-              title="Connexion sur ta nouvelle adresse"
+              title={
+                currentStep === 3
+                  ? 'Connecte toi sur ta nouvelle adresse'
```

Comment:
Un détail
```suggestion
                  ? 'Connecte-toi sur ta nouvelle adresse'
```
c'est plus français :D